### PR TITLE
added container to navbar and changed card margin from m-4 to mt-4

### PR DESCRIPTION
**Pull request by PranavBakre**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 __pycache__
 venv
-.vscode
\ No newline at end of file
+.vscode
+venv
```
Comment (prkprime) [inline .gitignore:4]: why venv double?

**Review by prkprime — CHANGES_REQUESTED** [review]
remove the duplicate venv entry from .gitignore file please

File: .gitignore (hunk 1 above)
Comment (PranavBakre) [reply .gitignore:4]: where double venv
it is .vscode and venv

**Review by prkprime — APPROVED** [review]
Thank You man.
It looks really nice now

**Pull request merged**